resolve_innerbuild.sh

```
@@ -0,0 +1,48 @@
+#/bin/bash
```

Shellcheck (0.11.0)

[error] Use #!, not just #, for the shebang.

(SC1113)